Editor/EditorCore/SmoothGroupEditor.cs

```diff
@@ -721,5 +721,13 @@ static Color32 GetDistinctColor(int index)
             new Color32(255, 255, 255, 255),    // White
             new Color32(0, 0, 0, 255),          // Black
         };
+
+        public static void Refresh()
```

Comment:
I'd use the existing `OnSelectionChanged` here, replacing the `Selection.selectionChanged += OnSelectionChanged;` earlier with the ProBuilder version - `MeshSelection.objectSelectionChanged`. I'm not sure that it is invoked when the strip action runs, although if it isn't it should be.